spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -65,51 +65,51 @@ You can also deploy Spring Boot applications to any Servlet 3.0+ compatible cont
 [[getting-started-installing-spring-boot]]
 == Installing Spring Boot
 Spring Boot can be used with "`classic`" Java development tools or installed as a command
-line tool. Regardless, you will need http://www.java.com[Java SDK v1.8] or higher. You
-should check your current Java installation before you begin:
+line tool. Either way, you need http://www.java.com[Java SDK v1.8] or higher. Before you
+begin, you should check your current Java installation by using the following command:
 
 [indent=0]
 ----
 	$ java -version
 ----
 
-If you are new to Java development, or if you just want to experiment with Spring Boot
-you might want to try the <<getting-started-installing-the-cli, Spring Boot CLI>> first,
+If you are new to Java development or if you want to experiment with Spring Boot,
+you might want to try the <<getting-started-installing-the-cli, Spring Boot CLI>> (Command Line Interface) first,
 otherwise, read on for "`classic`" installation instructions.
 
 
 
 [[getting-started-installation-instructions-for-java]]
-=== Installation instructions for the Java developer
-You can use Spring Boot in the same way as any standard Java library. Simply include the
+=== Installation Instructions for the Java Developer
+You can use Spring Boot in the same way as any standard Java library. To do so, include the
 appropriate `+spring-boot-*.jar+` files on your classpath. Spring Boot does not require
-any special tools integration, so you can use any IDE or text editor; and there is
-nothing special about a Spring Boot application, so you can run and debug as you would
+any special tools integration, so you can use any IDE or text editor. Also, there is
+nothing special about a Spring Boot application, so you can run and debug a Spring Boot program as you would
```

Comment:
The switch from "application" to "program" reads a little strangely to me. For consistency, shouldn't it be "application" again?

Reply:
"program" matches "any other Java program". That's the selling point in the sentence, so I thought it best to match "program".

Reply:
Got you. I still think it should be "application". A Spring Boot application is a particular type of Java program. I haven't heard of anyone refer to a Spring Boot program before. Application also ties in with the `@SpringBootApplication` annotation that pretty much every Boot app uses.

Reply:
OK. I'll put it back to application.